### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ekopalypse**
Add instructions on how to get a different version on Windows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ git clone https://github.com/vlang/sdl.git ~/.vmodules
 cd ~/.vmodules/sdl
 git checkout 2.0.12
 ```
+and for Windows (and then following the needed steps mentioned in the Windows specific section)
+```bash
+git clone https://github.com/vlang/sdl.git %HOMEPATH%/.vmodules/sdl
```
Comment (JalonSolov) [inline README.md:31]: This is what `v install sdl` does... might be better to stick with the standard command.

Comment (Ekopalypse) [reply README.md:31]: I think I don't really understand what you're getting at. If it does this on Unix, why was it described?
It doesn't work this way on Windows, tested on Windows 7 and Windows 10.
There, a "v install sdl" only clones the main branch, other branches are not available.

File: README.md [hunk 2]
```diff
@@ -26,6 +26,12 @@ git clone https://github.com/vlang/sdl.git ~/.vmodules
 cd ~/.vmodules/sdl
 git checkout 2.0.12
 ```
+and for Windows (and then following the needed steps mentioned in the Windows specific section)
```
Comment (Larpon) [inline README.md:29]:
```suggestion
and for Windows:
```

File: README.md [hunk 3]
```diff
@@ -26,6 +26,12 @@ git clone https://github.com/vlang/sdl.git ~/.vmodules
...
+```bash
+git clone https://github.com/vlang/sdl.git %HOMEPATH%/.vmodules/sdl
+cd %HOMEPATH%/.vmodules/sdl
+git checkout 2.0.12
+```
```
Comment (Larpon) [inline README.md:35]:
```suggestion
then follow the steps in the Windows section below.
```

**Comment by Larpon** [comment]
Thanks!

**Comment by Larpon** [comment]
If you push your changes to a branch instead of `master` it'll allow maintainers to cooperate on the branch.
In this case I could've done the changes directly and committed the PR. No biggie though, you fixed it quickly :slightly_smiling_face:

**Comment by Ekopalypse** [comment]
Sorry, I keep forgetting that :-(

**Pull request merged**